method/group.js

```diff
@@ -99,13 +99,9 @@ module.exports = function GroupMethods (ssb, keystore, state) {
         type: 'group/exclude-member',
         excludes: authorIds,
         tangles: {
-          members: {
-            root,
-            previous: [root] // TODO calculate previous for members tangle
-          },
-
+          members: { root, previous: [root] },
```

Comment:
GREAT! I am so glad that past me put this tangle in place. Makes this work so much easier (even if it's been "wrong")